types/aggregator.go

```diff
@@ -58,6 +60,7 @@ type AggregatorsParams struct {
 
 // UpdateAggregator request payload for updating an aggregator.
 type UpdateAggregator struct {
-	Name    *string `json:"name"`
-	Version *string `json:"version"`
+	Name          *string `json:"name"`
+	Version       *string `json:"version"`
+	EnvironmentID string  `json:"environment_id"`
```

Comment:
I propose passing a Environment string, validate is unique to the project and save the id in the database.